connector/mqtt/locust/src/scripts/generate_certs.py

```diff
@@ -0,0 +1,383 @@
+"""
+Certificate creation for Locust.
+"""
+
+import argparse
+from multiprocessing import Process
+import time
+import uuid
+import shutil
+import sys
+import os
+import redis
+import requests
+
+from src.ejbca.thing import Thing
+from src.config import CONFIG
+from src.utils import Utils
+
+
+LOGGER = Utils.create_logger("generate_cert")
+
+
+def generate_certs(ids: list) -> None:
+    """
+    Generates the certificates for the IDs.
+
+    Args:
+        ids: list of IDs.
+    """
+    start_time = time.time()
+
+    redis_conn = connect_to_redis()
+
+    for device_id in ids:
+        thing = Thing(CONFIG['app']['tenant'], device_id)
```

Comment:
when add a device on dojot a ejbca user is created. I noticed that Thing class will try to create it again.

Reply:
The new EJBCA does not create the user when a device was created.